release_tester/arangodb/instance.py

```diff
@@ -160,6 +169,7 @@ def resume_instance(self):
         else:
             logging.error("instance not available with this PID: " + str(repr(self)))
 
+    @step("Crash instance")
```

Comment:
this is a debug tool. can we make this red or orange?

Reply:
No, we cannot use custom statuses in allure. If this function is not meant to be used in test cases, it will not appear in the report anyway.

Reply:
No, unfortunately we cannot use custom statuses in allure.